pygmt/src/xyz2grd.py

```diff
@@ -14,14 +14,18 @@
 
 @fmt_docstring
 @use_alias(
+    A="duplicate",
+    D="dname",
     G="outgrid",
     I="spacing",
     R="region",
     V="verbose",
+    Z="onecolumn",
```

Comment:
Again, GMT.jl uses ['flags' for `-Z`](https://www.generic-mapping-tools.org/GMT.jl/v0.31/#GMT.xyz2grd), should we follow?

Reply:
`flags` is a bad name, because "an option" sometimes is also called "a flag".

Reply:
My preference would be 'z_convention', since this is what the *flags* provided in the argument is supposed to describe.